test/iron-dropdown.html

```diff
@@ -512,6 +526,50 @@
           });
         });
       });
+
+      suite('auto-cancel on blur', function() {
+
+        test('auto-cancel when focusable dropdown blurs', function(done) {
+          var elements = fixture('FocusableDropdownWithOtherFocusableElements');
+          var dropdown = Polymer.dom(elements).querySelector('iron-dropdown');
+          var content = Polymer.dom(dropdown).querySelector('.dropdown-content');
+          var button1 = Polymer.dom(elements).querySelector('#focusableButton1');
+          var button2 = Polymer.dom(elements).querySelector('#focusableButton2');
+
+          // Step 1: First button gets focus.
+          button1.addEventListener('focus', function() {
+            dropdown.open();
+            runAfterOpen(dropdown, function() {
```

Comment:
Good point, I've dropped that first button. The test now starts out with focus on the window, which is sufficient.